index.bs

```diff
@@ -201,9 +201,9 @@ The following steps are taken:
 <emu-alg>
 1. Let _loader_ be *this* value.
 1. If Type(_loader_) is not Object, throw a *TypeError* exception.
-1. If _loader_ does not have a [[Registry]] internal slot throw a *TypeError* exception.
 1. Return the result of transforming Resolve(_loader_, _name_, _referrer_) with a fulfillment handler that, when called with argument _key_, runs the following steps:
-  1. Return RequestReady(_loader_, _key_).
+  1. Let _entry_ be EnsureRegistered(_loader_, _key_).
+  1. Return _entry_.load().
```

Comment:
I'm torn about whether we want to directly call `.load()`, which could've been tampered with, versus calling some high-integrity abstract operation.

For an internal operation I'd feel strongly about this, but `.import` is a high-level method, so maybe it's fine for it just to be thought of as sugar for `_entry_.load()`.